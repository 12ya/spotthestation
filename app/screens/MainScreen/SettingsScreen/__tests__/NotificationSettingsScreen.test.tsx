import React from "react"
import { NotificationSettingsScreen } from "../NotificationSettingsScreen"
import { act, render } from "@testing-library/react-native"
import { NavigationContainer } from "@react-navigation/native"
import * as MockDate from "mockdate"

describe("NotificationSettingsScreen", () => {
  beforeEach(() => {
    MockDate.set("12-12-2012 10:10:10")
  })

  afterEach(() => {
    MockDate.reset()
  })

  it("renders correctly", async () => {
    const component = render(
      <NavigationContainer>
        <NotificationSettingsScreen />
      </NavigationContainer>,
    )

    await act(async () => {
      await new Promise((resolve) => setTimeout(resolve, 100))
    })

    expect(component.toJSON()).toMatchSnapshot()
  })
})
