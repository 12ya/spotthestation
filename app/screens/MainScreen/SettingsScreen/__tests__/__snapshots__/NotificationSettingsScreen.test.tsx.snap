// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <RCTScrollView
    accessibilityHint="Notification settings scrollable us area"
    accessibilityLabel="Notification settings scrollable us area"
    accessibilityRole="scrollbar"
    accessible={true}
    contentContainerStyle={
      Object {
        "flexGrow": 1,
        "paddingBottom": 86,
      }
    }
    scrollEnabled={true}
    style={
      Object {
        "paddingHorizontal": 52,
      }
    }
  >
    <View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
      >
        <View
          accessibilityHint="Navigates to the previous screen"
          accessibilityLabel="Back button"
          accessibilityRole="button"
          accessible={true}
          collapsable={false}
          focusable={true}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "paddingBottom": 16,
            }
          }
        >
          <View>
            <RCTImageView
              internal_analyticTag={null}
              resizeMode="contain"
              source={
                Array [
                  Object {
                    "height": undefined,
                    "uri": undefined,
                    "width": undefined,
                  },
                ]
              }
              style={
                Object {
                  "height": 35,
                  "overflow": "hidden",
                  "resizeMode": "contain",
                  "tintColor": "#DEDDDE",
                  "width": 35,
                }
              }
              tintColor="#DEDDDE"
            />
          </View>
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {},
                Array [
                  Object {
                    "fontSize": 23,
                    "lineHeight": 35,
                  },
                  Object {
                    "fontFamily": "Helvetica Neue",
                  },
                  Object {
                    "color": "#191015",
                  },
                ],
                undefined,
                undefined,
                Object {
                  "color": "#DEDDDE",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 26,
                  "lineHeight": 32,
                  "paddingBottom": 0,
                  "paddingLeft": 7,
                  "textAlign": "left",
                },
              ]
            }
          >
            settings.notificationSettingsData.backButton undefined
          </Text>
        </View>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {},
              Array [
                Object {
                  "fontSize": 23,
                  "lineHeight": 35,
                },
                Object {
                  "fontFamily": "Helvetica Neue",
                },
                Object {
                  "color": "#191015",
                },
              ],
              undefined,
              undefined,
              Object {
                "color": "#DEDDDE",
                "fontFamily": "Helvetica Neue",
                "fontSize": 52,
                "lineHeight": 63,
                "paddingBottom": 35,
                "textAlign": "left",
              },
            ]
          }
        >
          settings.notificationSettingsData.notificationTitle undefined
        </Text>
        <View
          style={
            Object {
              "borderBottomColor": "#1E2022",
              "borderBottomWidth": 2,
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 26,
              "paddingBottom": 26,
            }
          }
        >
          <View
            accessibilityHint="upcoming events notification switch"
            accessibilityLabel="upcoming events notification"
            accessibilityRole="text"
            accessible={true}
            style={
              Object {
                "width": "70%",
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {},
                  Array [
                    Object {
                      "fontSize": 23,
                      "lineHeight": 35,
                    },
                    Object {
                      "fontFamily": "Helvetica Neue",
                    },
                    Object {
                      "color": "#191015",
                    },
                  ],
                  undefined,
                  undefined,
                  Object {
                    "color": "#DEDDDE",
                    "fontFamily": "Helvetica Neue",
                    "fontSize": 35,
                    "lineHeight": 42,
                  },
                ]
              }
            >
              settings.notificationSettingsData.upcomingLabel undefined
            </Text>
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {},
                  Array [
                    Object {
                      "fontSize": 23,
                      "lineHeight": 35,
                    },
                    Object {
                      "fontFamily": "Helvetica Neue",
                    },
                    Object {
                      "color": "#191015",
                    },
                  ],
                  undefined,
                  undefined,
                  Object {
                    "color": "#ADADAE",
                    "fontFamily": "HelveticaNeue-Light",
                    "fontSize": 26,
                    "lineHeight": 32,
                  },
                ]
              }
            >
              settings.notificationSettingsData.upcomingTip undefined
            </Text>
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "marginTop": 15,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {},
                    Array [
                      Object {
                        "fontSize": 23,
                        "lineHeight": 35,
                      },
                      Object {
                        "fontFamily": "Helvetica Neue",
                      },
                      Object {
                        "color": "#191015",
                      },
                    ],
                    undefined,
                    undefined,
                    Array [
                      Object {
                        "color": "#ADADAE",
                        "fontFamily": "HelveticaNeue-Light",
                        "fontSize": 26,
                        "lineHeight": 32,
                      },
                      Object {
                        "color": "#0B3D91",
                      },
                    ],
                  ]
                }
              >
                settings.notificationSettingsData.customizeLabel undefined
              </Text>
            </View>
          </View>
          <View
            accessibilityHint="toggle upcoming events notifications"
            accessibilityLabel="switch button"
            accessibilityRole="switch"
            accessibilityState={
              Object {
                "checked": false,
                "disabled": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  Array [
                    Array [
                      Object {
                        "alignItems": "center",
                        "borderWidth": 3,
                        "flexDirection": "row",
                        "flexGrow": 0,
                        "flexShrink": 0,
                        "height": 35,
                        "justifyContent": "space-between",
                        "overflow": "hidden",
                        "width": 35,
                      },
                      Object {
                        "borderRadius": 23,
                        "borderWidth": 0,
                        "height": 40,
                        "width": 66,
                      },
                    ],
                    Object {
                      "backgroundColor": "#353739",
                    },
                    undefined,
                  ]
                }
              >
                <View
                  animatedStyle={
                    Object {
                      "value": Object {
                        "opacity": 0,
                      },
                    }
                  }
                  collapsable={false}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#0B3D91",
                      "borderColor": "rgba(0, 0, 0, 0)",
                      "height": "100%",
                      "opacity": 0,
                      "overflow": "hidden",
                      "paddingEnd": 6,
                      "paddingStart": 6,
                      "position": "absolute",
                      "width": "100%",
                    }
                  }
                />
                <View
                  animatedStyle={
                    Object {
                      "value": Object {
                        "marginStart": 6,
                        "start": "0%",
                      },
                    }
                  }
                  collapsable={false}
                  style={
                    Object {
                      "backgroundColor": "#F4F2F1",
                      "borderRadius": 17,
                      "height": 29,
                      "marginStart": 6,
                      "position": "absolute",
                      "start": "0%",
                      "width": 29,
                    }
                  }
                />
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "marginTop": 52,
                "width": "100%",
              },
              Object {
                "borderBottomColor": "#1E2022",
                "borderBottomWidth": 1,
                "marginTop": 10,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 26,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flex": 1,
                    "flexDirection": "row",
                  },
                  undefined,
                ]
              }
            >
              <Text
                accessibilityHint="settings.notificationSettingsData.notifyMeBefore"
                accessibilityLabel="title"
                accessibilityRole="text"
                accessible={true}
                allowFontScaling={false}
                style={
                  Array [
                    Object {},
                    Array [
                      Object {
                        "fontSize": 23,
                        "lineHeight": 35,
                      },
                      Object {
                        "fontFamily": "Helvetica Neue",
                      },
                      Object {
                        "color": "#191015",
                      },
                    ],
                    undefined,
                    undefined,
                    Array [
                      Object {
                        "color": "#ADADAE",
                        "fontFamily": "Helvetica Neue",
                        "fontSize": 19,
                        "lineHeight": 23,
                        "textTransform": "uppercase",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                settings.notificationSettingsData.notifyMeBefore undefined
              </Text>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={
              Object {
                "backgroundColor": "#1E2022",
                "borderRadius": 40,
                "height": 80,
                "justifyContent": "center",
                "marginBottom": 26,
                "overflow": "hidden",
              }
            }
          >
            <View
              accessibilityLabel="period select"
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "height": 35,
                  "justifyContent": "space-between",
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "fontSize": 16,
                    },
                    Array [
                      Object {
                        "color": "#DEDDDE",
                        "flex": 1,
                        "fontFamily": "Helvetica Neue",
                        "fontSize": 26,
                        "marginHorizontal": 17,
                        "paddingHorizontal": 0,
                        "paddingVertical": 0,
                        "textAlignVertical": "center",
                      },
                      Object {
                        "color": "#ADADAE",
                      },
                    ],
                    Object {},
                  ]
                }
              >
                settings.contactUsData.titlePlaceholder undefined
              </Text>
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "height": 80,
                    "justifyContent": "center",
                    "marginEnd": 35,
                  }
                }
              >
                <RCTImageView
                  internal_analyticTag={null}
                  resizeMode="contain"
                  source={
                    Array [
                      Object {
                        "height": undefined,
                        "uri": undefined,
                        "width": undefined,
                      },
                    ]
                  }
                  style={
                    Object {
                      "height": 40,
                      "overflow": "hidden",
                      "resizeMode": "contain",
                      "tintColor": "#ADADAE",
                      "width": 40,
                    }
                  }
                  tintColor="#ADADAE"
                />
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "borderBottomColor": "#1E2022",
                "borderBottomWidth": 2,
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 26,
                "paddingBottom": 26,
              },
              Object {
                "borderBottomWidth": 0,
                "marginTop": 22,
              },
            ]
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {},
                Array [
                  Object {
                    "fontSize": 23,
                    "lineHeight": 35,
                  },
                  Object {
                    "fontFamily": "Helvetica Neue",
                  },
                  Object {
                    "color": "#191015",
                  },
                ],
                undefined,
                undefined,
                Object {
                  "color": "#DEDDDE",
                  "fontFamily": "Helvetica Neue",
                  "fontSize": 35,
                  "lineHeight": 42,
                },
              ]
            }
          >
            settings.notificationSettingsData.privacyTitle undefined
          </Text>
          <View
            accessibilityHint="toggle privacy notifications"
            accessibilityLabel="switch button"
            accessibilityRole="switch"
            accessibilityState={
              Object {
                "checked": false,
                "disabled": undefined,
              }
            }
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flexDirection": "row",
                  },
                  undefined,
                ]
              }
            >
              <View
                style={
                  Array [
                    Array [
                      Object {
                        "alignItems": "center",
                        "borderWidth": 3,
                        "flexDirection": "row",
                        "flexGrow": 0,
                        "flexShrink": 0,
                        "height": 35,
                        "justifyContent": "space-between",
                        "overflow": "hidden",
                        "width": 35,
                      },
                      Object {
                        "borderRadius": 23,
                        "borderWidth": 0,
                        "height": 40,
                        "width": 66,
                      },
                    ],
                    Object {
                      "backgroundColor": "#353739",
                    },
                    undefined,
                  ]
                }
              >
                <View
                  animatedStyle={
                    Object {
                      "value": Object {
                        "opacity": 0,
                      },
                    }
                  }
                  collapsable={false}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#0B3D91",
                      "borderColor": "rgba(0, 0, 0, 0)",
                      "height": "100%",
                      "opacity": 0,
                      "overflow": "hidden",
                      "paddingEnd": 6,
                      "paddingStart": 6,
                      "position": "absolute",
                      "width": "100%",
                    }
                  }
                />
                <View
                  animatedStyle={
                    Object {
                      "value": Object {
                        "marginStart": 6,
                        "start": "0%",
                      },
                    }
                  }
                  collapsable={false}
                  style={
                    Object {
                      "backgroundColor": "#F4F2F1",
                      "borderRadius": 17,
                      "height": 29,
                      "marginStart": 6,
                      "position": "absolute",
                      "start": "0%",
                      "width": 29,
                    }
                  }
                />
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "marginTop": 52,
                "width": "100%",
              },
              Object {
                "marginTop": 0,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
                "marginBottom": 26,
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "flex": 1,
                    "flexDirection": "row",
                  },
                  undefined,
                ]
              }
            >
              <Text
                accessibilityHint="settings.notificationSettingsData.turnOffNotifications"
                accessibilityLabel="title"
                accessibilityRole="text"
                accessible={true}
                allowFontScaling={false}
                style={
                  Array [
                    Object {},
                    Array [
                      Object {
                        "fontSize": 23,
                        "lineHeight": 35,
                      },
                      Object {
                        "fontFamily": "Helvetica Neue",
                      },
                      Object {
                        "color": "#191015",
                      },
                    ],
                    undefined,
                    undefined,
                    Array [
                      Object {
                        "color": "#ADADAE",
                        "fontFamily": "Helvetica Neue",
                        "fontSize": 19,
                        "lineHeight": 23,
                        "textTransform": "uppercase",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                settings.notificationSettingsData.turnOffNotifications undefined
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "justifyContent": "space-between",
              }
            }
          >
            <View
              style={
                Object {
                  "width": "47%",
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {},
                    Array [
                      Object {
                        "fontSize": 23,
                        "lineHeight": 35,
                      },
                      Object {
                        "fontFamily": "Helvetica Neue",
                      },
                      Object {
                        "color": "#191015",
                      },
                    ],
                    undefined,
                    undefined,
                    Object {
                      "color": "#DEDDDE",
                      "fontFamily": "HelveticaNeue-Light",
                      "fontSize": 26,
                      "lineHeight": 32,
                      "marginBottom": 15,
                    },
                  ]
                }
              >
                settings.notificationSettingsData.from undefined
              </Text>
              <View
                accessibilityHint="open time picker"
                accessibilityLabel="from picker button"
                accessibilityRole="button"
                accessibilityState={
                  Object {
                    "disabled": true,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Array [
                      Object {
                        "alignItems": "center",
                        "borderRadius": 6,
                        "flexDirection": "row",
                        "justifyContent": "center",
                        "minHeight": 80,
                        "overflow": "hidden",
                        "paddingHorizontal": 17,
                        "paddingVertical": 17,
                      },
                      Object {
                        "backgroundColor": "#FFFFFF",
                        "borderColor": "#B6ACA6",
                        "borderWidth": 1,
                      },
                    ],
                    Array [
                      Object {
                        "backgroundColor": "#353739",
                        "borderRadius": 40,
                        "borderWidth": 0,
                        "height": 80,
                        "justifyContent": "space-between",
                        "paddingHorizontal": 23,
                        "width": "100%",
                      },
                      Object {
                        "opacity": 0.5,
                      },
                    ],
                    false,
                  ]
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {},
                      Array [
                        Object {
                          "fontSize": 23,
                          "lineHeight": 35,
                        },
                        Object {
                          "fontFamily": "Helvetica Neue",
                        },
                        Object {
                          "color": "#191015",
                        },
                      ],
                      undefined,
                      undefined,
                      Array [
                        Object {
                          "flexGrow": 0,
                          "flexShrink": 1,
                          "fontFamily": "HelveticaNeue-Medium",
                          "fontSize": 23,
                          "lineHeight": 29,
                          "textAlign": "center",
                          "zIndex": 2,
                        },
                        Object {
                          "color": "#FFFFFF",
                          "fontFamily": "Helvetica Neue",
                          "fontSize": 26,
                          "lineHeight": 30,
                        },
                        false,
                      ],
                    ]
                  }
                >
                  10:10 AM
                </Text>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "height": 80,
                      "justifyContent": "center",
                    }
                  }
                >
                  <RCTImageView
                    internal_analyticTag={null}
                    resizeMode="contain"
                    source={
                      Array [
                        Object {
                          "height": undefined,
                          "uri": undefined,
                          "width": undefined,
                        },
                      ]
                    }
                    style={
                      Object {
                        "height": 40,
                        "overflow": "hidden",
                        "resizeMode": "contain",
                        "tintColor": "#DEDDDE",
                        "width": 40,
                      }
                    }
                    tintColor="#DEDDDE"
                  />
                </View>
              </View>
            </View>
            <View
              style={
                Object {
                  "width": "47%",
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {},
                    Array [
                      Object {
                        "fontSize": 23,
                        "lineHeight": 35,
                      },
                      Object {
                        "fontFamily": "Helvetica Neue",
                      },
                      Object {
                        "color": "#191015",
                      },
                    ],
                    undefined,
                    undefined,
                    Object {
                      "color": "#DEDDDE",
                      "fontFamily": "HelveticaNeue-Light",
                      "fontSize": 26,
                      "lineHeight": 32,
                      "marginBottom": 15,
                    },
                  ]
                }
              >
                settings.notificationSettingsData.until undefined
              </Text>
              <View
                accessibilityHint="open time picker"
                accessibilityLabel="until picker button"
                accessibilityRole="button"
                accessibilityState={
                  Object {
                    "disabled": true,
                  }
                }
                accessible={true}
                collapsable={false}
                focusable={true}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Array [
                    Array [
                      Object {
                        "alignItems": "center",
                        "borderRadius": 6,
                        "flexDirection": "row",
                        "justifyContent": "center",
                        "minHeight": 80,
                        "overflow": "hidden",
                        "paddingHorizontal": 17,
                        "paddingVertical": 17,
                      },
                      Object {
                        "backgroundColor": "#FFFFFF",
                        "borderColor": "#B6ACA6",
                        "borderWidth": 1,
                      },
                    ],
                    Array [
                      Object {
                        "backgroundColor": "#353739",
                        "borderRadius": 40,
                        "borderWidth": 0,
                        "height": 80,
                        "justifyContent": "space-between",
                        "paddingHorizontal": 23,
                        "width": "100%",
                      },
                      Object {
                        "opacity": 0.5,
                      },
                    ],
                    false,
                  ]
                }
              >
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {},
                      Array [
                        Object {
                          "fontSize": 23,
                          "lineHeight": 35,
                        },
                        Object {
                          "fontFamily": "Helvetica Neue",
                        },
                        Object {
                          "color": "#191015",
                        },
                      ],
                      undefined,
                      undefined,
                      Array [
                        Object {
                          "flexGrow": 0,
                          "flexShrink": 1,
                          "fontFamily": "HelveticaNeue-Medium",
                          "fontSize": 23,
                          "lineHeight": 29,
                          "textAlign": "center",
                          "zIndex": 2,
                        },
                        Object {
                          "color": "#FFFFFF",
                          "fontFamily": "Helvetica Neue",
                          "fontSize": 26,
                          "lineHeight": 30,
                        },
                        false,
                      ],
                    ]
                  }
                >
                  10:10 AM
                </Text>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "height": 80,
                      "justifyContent": "center",
                    }
                  }
                >
                  <RCTImageView
                    internal_analyticTag={null}
                    resizeMode="contain"
                    source={
                      Array [
                        Object {
                          "height": undefined,
                          "uri": undefined,
                          "width": undefined,
                        },
                      ]
                    }
                    style={
                      Object {
                        "height": 40,
                        "overflow": "hidden",
                        "resizeMode": "contain",
                        "tintColor": "#DEDDDE",
                        "width": 40,
                      }
                    }
                    tintColor="#DEDDDE"
                  />
                </View>
              </View>
            </View>
          </View>
        </View>
        <
          date={2012-12-12T08:10:10.000Z}
          isVisible={false}
          mode="time"
          onCancel={[Function]}
          onConfirm={[Function]}
        />
        <
          date={2012-12-12T08:10:10.000Z}
          isVisible={false}
          mode="time"
          onCancel={[Function]}
          onConfirm={[Function]}
        />
      </View>
    </View>
  </RCTScrollView>
</div>
`;
